File: examples/linreg/example_beta.R
library("mmstat4")
cps78_85 <- ghload("cps.rds")
xs  <- subset(cps78_85, year==85, c("lwage", "educ"))
lms <- lm (scale(lwage)~scale(educ), data=xs)
summary(lms)
#
library("QuantPsyc")
lm <- lm (lwage~educ, data=xs)
lm.beta(lm)

File: examples/linreg/example_beta.py
import numpy as np
import pandas as pd
import statsmodels.formula.api as smf
from sklearn.preprocessing import StandardScaler
cps78_85 = r['mmstat4::ghload']("cps.rds")
cps    = cps78_85[cps78_85['year'] == 85]
xs     = cps[['lwage', 'educ']]
scaler = StandardScaler()
xs_scaled = pd.DataFrame(scaler.fit_transform(xs), columns=['lwage', 'educ'])
lms = smf.ols('lwage ~ educ', data=xs_scaled).fit()
print(lms.summary())
